package.json

```diff
@@ -253,7 +253,6 @@
     "lint-staged": "^11.0.0",
     "lockfile-lint": "^4.9.6",
     "nock": "11.8.0",
-    "proxy": "^2.1.1",
     "proxyquire": "^1.8.0",
```

Comment:
With the addition of `test/lib/proxy-server.js`, we no longer need this dependency.